### ISSUE TITLE
Access point doesn't work with password

### ISSUE THREAD

**Issue by ayyagari** [tessel/t2-cli#1004]
I tried `t2 ap -n myssid -p mypassword` and myssid doesn't show up at all in access points list in my computer/mobile phone wifi list. If I just used `t2 ap -n otherssid' then otherssid is visible in wifi list and lets my computer connect to it.
How to make tessel 2 and cli work for an ap with password?
Environment: 

```
INFO Tessel [myt2] CLI version: 0.0.29
INFO Tessel [myt2] Firmware version: 0.0.16
INFO Tessel [myt2] Node version: 4.5.0
```

**Comment by HipsterBrown** [comment]
@ayyagari Sorry to hear you're running into some issues with the `ap` setup. I've not had an issue with setting a password when creating a new access point yet but I will test that configuration out on my Tessel tonight and keep you updated. 

When you try `t2 ap -n myssid -p mypassword`, does the orange/amber light come on after the success message? Also, is your Tessel already connected to wifi when creating an access point?

**Comment by johnnyman727** [comment]
I would also check to make sure you are using a valid psk2 password. I believe it has to be something like 8 or more characters.

**Comment by HipsterBrown** [comment]
@ayyagari Are you still experiencing issues with creating an access point with a password?

**Comment by bmccord** [comment]
I am having the same problem.  So far, here is what I have found:

1.  If I leave the password and security off, everything is fine.  Phone can attach, etc.  So, this works fine: t2 ap -n myssid
2.  If I provide a password (I have tried several easy and complex to make sure psk2 is satisfied), it says everything is fine.  But, I get no orange lights on the Tessel and it doesn't show up on my list of wifi devices on my phone.
3.  I have tried it with the wifi on and off.  Neither work.
4.  Running the command in point 1 again at this point turns the orange light back on and makes it available to my phone...but it doesn't have any security.

Hope this helps.

**Comment by HipsterBrown** [comment]
@bmccord Thanks for those steps. I'll start investigating this issue again this week.

**Comment by rungkutta** [comment]
I've seen the same issue.  I'm not sure exactly what's going on, but I have noticed that the t2 ap command doesn't assign a password to the access point's wifi-iface 'key' parameter in /etc/config/wireless.  It does set the SSID and encryption method.  I suspect that having an encryption method set but no password is why the access point doesn't work.

I can SSH to the tessel and set the password manually with the same 'uci set' command that commands.js uses.  By doing this, I can get the access point to work with encryption/security.

Incidentally, I've also noticed that in messing around with the 't2 ap' command, several new empty 'wifi-iface' sections have been added to /etc/config/wireless.  I suspect there's some problem with the create vs update logic in access-point.js, but it's not immediately obvious to me what it is.

**Comment by HipsterBrown** [comment]
Thanks for letting us know @rungkutta. I'll check out those empty `wifi-iface` sections and the issue with `password` not being set on the correct sections.

**Comment by HipsterBrown** [comment]
@rungkutta @ayyagari @bmccord thank you for your patience. This has been fixed in version `0.1.4` -> https://github.com/tessel/t2-cli/releases/tag/v0.1.4

**Comment by bmccord** [comment]
Excellent!  Thank you!
